docs/integrations/gitlab.md

```diff
@@ -0,0 +1,74 @@
+
+First, navigate to the **Integrations** page for the appropriate resource:
+- To configure a GitLab integration for your **Developer Account**, click the double arrow button from the navigation at the top of the page, select your account from the dropdown, and then select the **Integrations** tab to manage integrations for your account.
+- To configure a GitLab integration for your **Organization**, click the double arrow button from the organization switcher at the top of the page and select the organization from the dropdown. Once you've selected your organization, go to the **Integrations** tab to manage the integrations for the organization.
+- To configure a GitLab integration for your **Tenant**, click the double arrow button from the tenant switcher at the top of the Pipes console, select your tenant, and then select **Tenant Settings**. This option will only be visible in a custom tenant for which you are a [tenant owner](/pipes/docs/accounts/tenant/people#tenant-roles).  Once you've selected your tenant, go to the **Integrations** tab to manage the integrations for the tenant.
+
+Next, click the **New Integration** button. You will be asked to select the type of integration to be created.
+
+![](/images/docs/pipes/integrations_new_gitlab.png)
+
+Select **GitLab** and click **Next**.
+
+GitLab Integration configuration management form shows up. Enter the following information:
+- **Handle** for the integration. This handle should be meaningful and must be unique for all integrations in the tenant (including any org-level integrations).
+- **Host** for your GitLab installation which defaults to **gitlab.com**.
+- **Token** to be used by Pipes to discover projects. Please note that the token must be granted `api` access which is required to create and manage webhooks that enables Pipes to keep the mods managed by the integration up-to-date. Pipes supports [user](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html), [group](https://docs.gitlab.com/ee/user/group/settings/group_access_tokens.html) or [project](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html) access tokens.
+- **Token Scope** is an optional section which allows you to further scope a token down to a GitLab group / project. The following cases might require you to configure this section:
+    - Using a **Personal Access Token** you need to mention the group / project to be used by Pipes to setup the webhook against.
+    - Using a **Group Access Token** and you want to further scope the webhook creation down to a group / project and restrict list of projects accessible to Pipes.
+
+![](/images/docs/pipes/integrations_gitlab_config.png)
+
+Click on **Create** after entering the appropriate information. You can also optionally click on **Test** to verify whether your configuration is accurate before creating the integration.
```

Comment:
Put the bit about optionally testing before the part about clicking Create.  
Then afterward, perhaps a sentence about what to expect next.   What does it do? Will I have to wait? Will everything show up instantly, or will the discovery take some time...

Reply:
Done